src/components/Menu/Menu.tsx

```diff
@@ -14,11 +14,25 @@ const Menu: React.FC<{}> = () => {
       chrome.tabs.sendMessage(tabs[0].id, { type: Messages.NEW_NOTE });
     }
   };
+
+  const deleteAllNotes = async () => {
+    const tabs = await chrome.tabs.query({
+      active: true,
+      currentWindow: true,
+    });
+
+    if (tabs.length > 0) {
+      chrome.tabs.sendMessage(tabs[0].id, { type: Messages.DELETE_ALL });
+    }
+  };
   return (
     <StyledList>
       <StyledListItem onClick={handleClick}>
         <StickyNote2Icon fontSize="small" /> Create a new note
       </StyledListItem>
+      <StyledListItem onClick={deleteAllNotes}>
+        <StickyNote2Icon fontSize="small" /> Delete all notes from page
```

Comment:
we might want another icon?

Reply:
It's just a helper for us, we might delete it later